src/main/resources/data/geodes/worldgen/configured_feature/certus_geode.json

```diff
@@ -0,0 +1,120 @@
+{
+	"type": "minecraft:geode",
+	"config": {
+		"blocks": {
+			"filling_provider": {
+				"type": "minecraft:simple_state_provider",
+				"state": {
+					"Name": "minecraft:water"
+				}
+			},
+			"inner_layer_provider": {
+				"type": "minecraft:simple_state_provider",
+				"state": {
+					"Name": "ae2:quartz_block"
+				}
+			},
+			"alternate_inner_layer_provider": {
```

Comment:
The middle layer and alternate inner layer providers do not match the 1.19-1.19.2 version, is this intended, or should those also be updated?

Reply:
Somehow I realized having it the same way in the 1.19-1.19.2 version made too many budding blocks generate, thus the difference. Also I may have been a little lazy around the corners. But tbh, I don't think that's really a problem considering how easy it is to get budding blocks out of a few certus quartz with AE2